1-exercises/B-setting-values/exercise1.js

```diff
@@ -23,6 +23,11 @@ let capitalCities = {
 */
 
 // write code here
+capitalCities.UnitedKingdom.population = 8980000;
+capitalCities.China.population = 21500000;
+capitalCities.Peru = {};
+capitalCities.Peru.name = "Lima";
+capitalCities.Peru.population = 9750000;
```

Comment:
These lines could be condensed as one i.e. assign an object with name and population keys at line 28.